mne_connectivity/vector_ar/var.py

```diff
@@ -151,10 +158,31 @@ def vector_auto_regression(
     # 1. determine shape of the window of data
     n_epochs, n_nodes, _ = data.shape
 
+    cv_alphas = None
+    if isinstance(l2_reg, str):
+        if l2_reg == 'auto':
+            
+            # determine condition of matrix across all epochs
+            conds = np.linalg.cond(data)
+            if np.any(conds > 1e6):
+                # matrix is rank-deficient, so regularization must be used with
+                # cross-validation alphas values
+                cv_alphas = np.logspace(-15,5,11)
+                
+                # TODO: Add message letting user know that matrix is ill-conditioned
+                # and the above alpha set will be searched
```

Comment:
We could prolly just use `warn(<insert message>)`

and then add an import message at the top for `from mne.utils import warn`, which will play nicely w/ the rest of MNE settings (https://mne.tools/dev/generated/mne.utils.warn.html).